extensions/sql-database-projects/src/controllers/projectController.ts

```diff
@@ -1451,7 +1502,8 @@ export class ProjectsController {
 				newProjName: projectInfo.projectName,
 				folderUri: vscode.Uri.file(projectInfo.outputFolder),
 				projectTypeId: constants.emptySqlDatabaseProjectTypeId,
-				sdkStyle: !!options?.isSDKStyle
+				sdkStyle: !!options?.isSDKStyle,
+				configureDefaultBuild: true //TODO: sai
```

Comment:
What is this TODO for? If it's for a future change, please add link to issue

Reply:
Thanks for identifying it, it should be removed!